SECP-service/src/test/java/com/visucius/secp/Controllers/UserRegistrationControllerTest.java

```diff
@@ -95,12 +117,31 @@ public void LastNameIsEmptyTest()
         assertEquals(response.getMessage(), UserErrorMessage.USER_NOT_CREATED);
     }
 
+    @Test
+    public void DisplayNameIsEmptyTest()
+    {
+        UserRegistrationRequest request = new UserRegistrationRequest(
+            "ali",
+            "farah",
+            "alifarah",
+            "",
+            "test@gmail.com",
+            "Password1");
```

Comment:
You could've made 1 method to create your request (requiring a parameter if needed) instead of repeating it but too late :)